src/System.Device.Ports/System.Device.Ports.SerialPort/InternalResources.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Device.Ports.SerialPort.Resources;
+using System.IO;
+
+using Marshal = System.Runtime.InteropServices.Marshal;
```

Comment:
NIT: why do we need to use aliases?

Reply:
I just got the code from the original SerialPort class, no particular reason.